Source/Core/Core/PowerPC/JitArm64/JitArm64_LoadStore.cpp

```diff
@@ -751,6 +753,31 @@ void JitArm64::stmw(UGeckoInstruction inst)
     gpr.Unlock(addr_base_reg);
 }
 
+void JitArm64::WriteBranchWatch_dcbx(u32 origin, u32 destination, UGeckoInstruction inst,
+                                     Arm64Gen::ARM64Reg n, BitSet32 gprs_in_use,
+                                     BitSet32 fprs_in_use)
+{
+  // It's impossible to do the ARM64Reg::X0 move skip optimization that JitArm64::WriteBranchWatch
+  // does, because n will be in ARM64Reg::W0 in the context of JitArm64::dcbx!
+  const ARM64Reg WA = gpr.GetReg(), branch_watch = EncodeRegTo64(WA);
+  const ARM64Reg WB = gpr.GetReg();
+  MOVP2R(branch_watch, &m_branch_watch);
+  LDRB(IndexType::Unsigned, WB, branch_watch,
+       static_cast<int>(Core::BranchWatch::GetOffsetOfRecordingActive()));
+  FixupBranch branch = TBNZ(WB, true);
+  ABI_PushRegisters(gprs_in_use);
+  m_float_emit.ABI_PushRegisters(fprs_in_use, WA);
+  ABI_CallFunction(
+      m_ppc_state.msr.IR ? &Core::BranchWatch::HitV_fk_n : &Core::BranchWatch::HitP_fk_n,
+      branch_watch, Common::BitCast<u64>(Core::FakeBranchWatchCollectionKey{origin, destination}),
+      inst.hex, n);
+  ABI_PopRegisters(gprs_in_use);
+  m_float_emit.ABI_PopRegisters(fprs_in_use, WB);
+  gpr.Unlock(WA);
+  gpr.Unlock(WB);
+  SetJumpTarget(branch);
```

Comment:
```suggestion
  SetJumpTarget(branch);
  gpr.Unlock(WA);
  gpr.Unlock(WB);
```

In practice I think the emitted code will be okay even if you don't reorder these, but the code will be easier to reason about if you reorder them. Same for the other variants of this function. (Or well, I suppose this comment might not be relevant anymore once you fix what I said about not calling `Lock` inside conditionally executed code...)